mod/wet4/views/default/page/elements/login_menu.php

```diff
@@ -195,18 +195,24 @@
 
 <script>
 $(document).ready(function () {
-$('#signin-link').click(function () {
-    $('#signin-dropdown').toggle();
-    $('#session').toggleClass('active');
-    return false;
-});
-$('#signin-dropdown').click(function(e) {
-    e.stopPropagation();
-});
-$(document).click(function() {
-    $('#signin-dropdown').hide();
-    $('#session').removeClass('active');
-});
+    $('#signin-link').click(function () {
+        $('#signin-dropdown').toggle();
+        $('#session').toggleClass('active');
+        return false;
+    });
+
+    $('#signin-dropdown').click(function(e) {
+        e.stopPropagation();
+    });
+
+    $(document).click(function() {
+        $('#signin-dropdown').hide();
+        $('#session').removeClass('active');
+    });
+
+    $('#login_focus').click(function(){
```

Comment:
@piet0024  I add a focus when user press ENTER on the login link. Also, I change the focus function to the id signin-link